db.go

```diff
@@ -15,6 +16,7 @@ type DB struct {
 	Conns      map[string]*gorm.DB
 	Migrations map[string]map[string]Table
 	Seeders    map[string]map[string]any
```

Comment:
Khusus untuk **Migrations** dan **Seeders** map tidak perlu mutex locking, karena implementasi di server idealnya hanya mengeksekusi migration dan seeder pada **single server in single call** di awal start server